guides/assets/stylesrc/components/_code-container.scss

```diff
@@ -22,7 +22,8 @@ intermediate whitespace looks uniform. */
 
   // Padding and spacing for LTR vs RTL langauge defaults
   :where(body[dir="ltr"]) & { background-position: 10px 10px; padding-right: 1em !important; padding-left: 56px !important; }
-  :where(body[dir="rtl"]) & { background-position: calc(100% - 10px) 10px; padding-right: 56px !important; padding-left: 1em !important; }
+  :where(body[dir="rtl"]) &,
+  html.translated-rtl & { background-position: calc(100% - 10px) 10px; padding-right: 56px !important; padding-left: 1em !important; }
```

Comment:
We can probably the `.translated-rtl` from here as well, and switch from `body` to `html` on the declarations above.

Reply:
Fixed in 196635c3